packages/@glimmer/syntax/lib/v2-a/loose-resolution.ts

```diff
@@ -76,7 +76,7 @@ export function AppendSyntaxContext(node: ASTv1.MustacheStatement): ASTv2.FreeVa
       ? ASTv2.LooseModeResolution.trustingAppend({ invoke: isInvoke })
       : ASTv2.LooseModeResolution.append({ invoke: isInvoke });
   } else {
-    return isInvoke ? ASTv2.STRICT_RESOLUTION : ASTv2.LooseModeResolution.fallback();
+    return ASTv2.LooseModeResolution.fallback();
```

Comment:
Hmm, does this introduce property fallback behavior?

Reply:
Chatted with @pzuraq, we already have tests confirming implicit this fallback does **not** happen for strict.

@pzuraq - if you have a chance, would you mind linking to that test?

Reply:
In Ember, this previously would have been compiled to `{{component foo.bar baz=123}}` or something like that, so I think the ship has sailed in that regard. That's why I included the change here.